app/helpers/users_helper.rb

```diff
@@ -49,6 +49,7 @@ def options_for_user_neighbourhoods(for_user)
   def options_for_tags
     policy_scope(Tag)
       .select(:name, :type, :id)
+      .where(type: 'Partnership')
```

Comment:
didn't work, we would need to write a `app/policies/partnership_policy.rb` I think

Reply:
hmm. weird. i thought rails magic would cover this with some kind of `as_partnership` magic scope.

okay i would suggest then adding a scope on the Tag model like
```ruby
scope(:partnerships) { where(type: 'Partnership') }
```
as this will keep the STI inheritance type complexity in one place in the Tag model and just use that scope externally